sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Customized/Models/AggregateSeries.cs

```diff
@@ -47,12 +47,12 @@ internal partial class AggregateSeries
         /// Interval size used to group events by.
         /// </param>
         /// <exception cref="ArgumentNullException">
-        /// <paramref name="timeSeriesId"/> or <paramref name="searchSpan"/> is null.
+        /// <paramref name="searchSpan"/> is null.
         /// </exception>
         public AggregateSeries(TimeSeriesId timeSeriesId, DateTimeRange searchSpan, TimeSpan interval)
         {
-            TimeSeriesId = timeSeriesId ?? throw new ArgumentNullException(nameof(timeSeriesId));
             SearchSpan = searchSpan ?? throw new ArgumentNullException(nameof(searchSpan));
+            TimeSeriesId = timeSeriesId;
```

Comment:
And now not validating for null?

Reply:
No. Because TimeSeriesId is a struct now, so it's value based as opposed to reference based. So it cannot be null.